### DISCUSSION TITLE
¿Is my printer bed to big?

### DISCUSSION THREAD

**Discussion by 3D2sh** [Willmac16/OctoPrint-GcodeLeveling#18] [Q&A]
Hello every one, I have a little question for you, i´ve been using this pluging for at least 2 years, and I and i've never had a problem, but recently  I have tried to print on the entire bed and I have some problems to achieve it, my printing area is 60cmx60cm.

What happens is that my printer has a deformed bed in the shape of a extruded U, where the sides are higher than the central part, and when I want to print a line from side to side, only at the highest sides it prints well, but as it approaches to the central part, the extruder dont go down enough, does anyone have any idea why is this happening?

This does not happen with small pieces, I can print these pieces anywhere on the table and print them without problems, I mean 20cmx20cm for example, the problem comes as I already said when i try to use the entire bed.

Does anyone have any idea why is this happening?

**Comment by Willmac16** [comment]
Hi @3D2sh! I'm glad to hear you've been using my plugin for the past two years!

Could you send me a screenshot of your GcodeLeveling Config so I can better diagnose the issue. My guess is more points and a higher degree polynomial would solve the problem, but I'll get you a better answer with the screenshots.

**Reply by 3D2sh** [reply]
Yes actually your code help me alot, man so thank u very much for creat it and for answer me <3


![Gcode 1](https://user-images.githubusercontent.com/120670864/207947747-f3845bff-2a98-4533-9ca5-fc8ffd894a53.png)
![Gcode 2](https://user-images.githubusercontent.com/120670864/207947874-0ef94dad-cae2-447f-a10b-14f1ec563c94.png)
![Gcode 3](https://user-images.githubusercontent.com/120670864/207947881-fb6f59e8-92a4-4dd5-8954-df7b7b8c2a0f.png)
![Gcode 4](https://user-images.githubusercontent.com/120670864/207947886-6653ec52-7c64-433c-9312-6d6104002b3f.png)

**Reply by Willmac16** [reply]
That looks like a lot of deflection, so I would start by gathering more points. Probably a 5x5 grid, but more accurate points can't hurt. I would try to measure each point to more precision than a tenth of a millimeter. Aim for a Z change when you test of 0.01mm. If that doesn't seem to help you could try bumping the Surface Complexity X up to 4.

**Reply by Willmac16** [reply]
Also, if you're willing to share, I'd love to see your printer because 60x60cm sounds super cool.

**Reply by 3D2sh** [reply]
thanks for ur advice, I will try to put it on action and I will send you my advances, and of course, my printer is a TENLOG D6pro, actually is a very uncommmon printer, but it make the work, i been having a lot of troubles because the design of the printer is not the best and the problem is the size it, no just in the bed but with every system, i recommend it but if you ever buy one prepare your self for a big challenge. 
![WhatsApp Image 2022-12-15 at 3 
![WhatsApp Image 2022-12-15 at 3 55 55 PM (1)](https://user-images.githubusercontent.com/120670864/207975561-6e4a6520-c2b1-4e94-b43c-3e1b902de38e.jpeg)
55 55 PM](https://user-images.githubusercontent.com/120670864/207975546-a2af07c2-7145-4c58-bf62-8e6332e60b07.jpeg)

**Reply by 3D2sh** [reply]
![WhatsApp Image 2022-12-15 at 3 55 55 PM (1)](https://user-images.githubusercontent.com/120670864/207975767-967bac26-e975-4fb5-8f47-3bb882ccf0cd.jpeg)
